TabloidMVC/Views/Post/MyIndex.cshtml

```diff
@@ -0,0 +1,85 @@
+﻿@model IEnumerable<TabloidMVC.Models.Post>
+
+@{
+    ViewData["Title"] = "MyIndex";
+}
+
+<h1>MyIndex</h1>
+
+<p>
+    <a asp-action="Create">Create New</a>
+</p>
+<table class="table">
+    <thead>
+        <tr>
+            @*<th>
+            @Html.DisplayNameFor(model => model.Id)
+            </th>*@
+            <th>
+                @Html.DisplayNameFor(model => model.Title)
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.UserProfile)
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.Category)
+            </th>
+            @*   <th>
+            @Html.DisplayNameFor(model => model.Content)
+            </th>*@
```

Comment:
We shouldn't have any commented out code in production-- if you don't need it, you can just delete it! You can always add stuff back in later if you decide you need it

Reply:
Done!